src/main/resources/db/changelog/007_Add_table_applications.sql

```diff
@@ -0,0 +1,15 @@
+-- liquibase formatted sql
+-- changeset author:oskachkov
+
+CREATE TABLE applications(
+    id SERIAL,
+    amount INTEGER NOT NULL ,
+    status status_enum NOT NULL DEFAULT 'new',
+    created TIMESTAMP NOT NULL DEFAULT NOW(),
+    assigned TIMESTAMP NOT NULL ,
```

Comment:
I suppose assigned can be null if application in status 'new' and advisor_id is null